# Ranging measurement message from Decawave EVK1000 eval kit.
# Tag to Anchor distance in m.

std_msgs/Header header
float32 distance_cor  # bias correction has been applied
float32 distance_raw
